src/core/Akka.Cluster.Tests.MultiNode/ClusterDeathWatchSpec.cs

```diff
@@ -117,7 +117,7 @@ public void ClusterDeathWatchSpecTests()
 
         public void An_actor_watching_a_remote_actor_in_the_cluster_must_receive_terminated_when_watched_node_becomes_down_removed()
         {
-            Within(TimeSpan.FromSeconds(20), () =>
+            Within(TimeSpan.FromSeconds(30), () =>
```

Comment:
I could change this back to 20 - changed it when I was testing some issues that turned out to be the fault of the MNTR.